cl/tests/cases.py

```diff
@@ -179,6 +180,21 @@ def tearDown(self) -> None:
         self.restart_celery_throttle_key()
         super().tearDown()
 
+    def assert_es_feed_content(self, node_tests, response, namespaces):
```

Comment:
A comment here would be helpful, please.

Reply:
Sure. Comment added.